lib/shoryuken/middleware/server/exponential_backoff_retry.rb

```diff
@@ -0,0 +1,54 @@
+module Shoryuken
+  module Middleware
+    module Server
+      class ExponentialBackoffRetry
+        include Util
+
+        def call(worker, queue, sqs_msg, body)
+          started_at = Time.now
```

Comment:
When I tested it at the console, it seemed that the visibility timeout actually starts at each receipt of the message.  I was able to set a visibility timeout of 43180 on a message that was first received 10 minutes ago, because the most recent receipt was 10 seconds ago.